### PR TITLE
Syndicate uniform supplies

### PR DESCRIPTION
**Pull request by heepox**
For when you want to look stylishly evil.

## About The Pull Request

Adds a Syndicate Uniform Supplies Crate Containing some Stylish Syndicate Gear
Tactical Turtlenecks
Combat Boots
Syndicate Berets
Armored Vests (no syndicate Equivalent sadly)
Combat Gloves
Balaclavas (Still cool might change to Syndie masks but i dont want them to look totally like nukies)

## Why It's Good For The Game

Adds the ability for someone to order Syndicate Clothes and some armor from an Emagged Cargo console for 3000 credits to equip themselves and 2 others in Fashionably evil clothing.

## Changelog
:cl:
add: Syndicate Uniform Supplies crate to EMAG:ed cargo console containing three sets of syndicate clothing and equipment to wear. Costs 3000.
/:cl:

### CODE REVIEW

**Comment by ItsTisme** [comment]
Id make them more expensive purely because of combat gloves being insulated. Other then that looks fun

**Comment by BrumalLake** [comment]
the contraband pack and smugglers already have syndicate masks
save the people a few bucks will you

**Comment by williamcll** [comment]
Can you put this on the nukie outpost as well?

**Comment by Boxlta** [comment]
add syndicate admirial equipment to a second crate for like, 5000

**Comment by williamcll** [comment]
> add syndicate admirial equipment to a second crate for like, 5000

Pretty sure it's called strategist.

**Comment by Howluinb** [comment]
Do I misunderstand what you mean by Armored Vests, or do you mean stuff like Armored greatcoat/overcoat ect ?
Those are literally way too strong to be buyable

**Comment by BrumalLake** [comment]
armored vest as in mk IV swat hardsuit
yes, armored vest exactly as written in the code
aand you don't even need an electromagnetic sequencer to buy that stuff anyways

**Comment by heepox** [comment]
> armored vest as in mk IV swat hardsuit
> yes, armored vest exactly as written in the code
> aand you don't even need an electromagnetic sequencer to buy that stuff anyways

yes its the ones sec and barman have. Literally called that in code. Its there for convenience. and you dont need an emag to buy it but you need sec access to open if you dont have emag.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -274,6 +274,32 @@
 	crate_name = "emergency crate"
 	crate_type = /obj/structure/closet/crate/internals
 
+/datum/supply_pack/emergency/syndieclothes
+	name = "Syndicate Uniform Supplies"
+	desc = "(*!&@#SAD ABOUT THAT NULL_ENTRY, HUH OPERATIVE? WELL, THIS LITTLE ORDER WILL MAKE YOU STYLISH SYNDICATE STYLE. CONTAINS A COLLECTION OF THREE TACTICAL TURTLENECKS, THREE COMBAT BOOTS, THREE COMBAT GLOVES, THREE BALACLAVAS, THREE SYNDICATE BERETS AND THREE ARMOR VESTS!#@*$"
```
Comment (Archanial) [inline code/modules/cargo/packs.dm]: don't mention OOC stuff IC

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -274,6 +274,32 @@
...
+	hidden = TRUE
+	cost = 1800
```
Comment (Archanial) [inline code/modules/cargo/packs.dm]: 1800 is too low

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (Archanial) [reply code/modules/cargo/packs.dm]: this is still a contraband and is significantly harder to ship to the station. If anything it should cost more to sponsor syndicate.

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (heepox) [reply code/modules/cargo/packs.dm]: literally copied from the other emag crate.

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (heepox) [reply code/modules/cargo/packs.dm]: other crate has actyally useful stuff... if anything this should be cheaper.

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Archanial) [reply code/modules/cargo/packs.dm]: This doesn't justify anything. Just delete SAD NULL_ENTRY

File: code/modules/cargo/packs.dm (hunk 2 above)
Comment (Archanial) [reply code/modules/cargo/packs.dm]: Those are illegally shipped syndicate goods, it doesn't matter if they're useful, they're hard to ship.

Comment (Archanial) [reply code/modules/cargo/packs.dm]:
```suggestion
	cost = 3000
```

**Review by Archanial — APPROVED** [review]

**Pull request merged**